docroot/themes/humsci/humsci_airy/humsci_airy.info.yml

```diff
@@ -0,0 +1,18 @@
+name: HumSci Airy
+type: theme
+base theme: humsci_basic
```

Comment:
also checked drupal docs, and `base theme` with a space is apparently correct 🤦‍♀ https://www.drupal.org/docs/8/theming-drupal-8/defining-a-theme-with-an-infoyml-file